compiler/src/dmd/cparse.d

```diff
@@ -4234,7 +4236,8 @@ final class CParser(AST) : Parser!AST
             {
                 if (specifier.alignasExp)
                     error(specifier.alignasExp.loc, "no alignment-specifier for bit field declaration"); // C11 6.7.5-2
-                auto s = new AST.BitFieldDeclaration(width.loc, dt, id, width);
+                AST.Dsymbol s = new AST.BitFieldDeclaration(width.loc, dt, id, width);
+                s = applySpecifier(s, specifier);
```

Comment:
Applies packed to bitfields, same is already done for regular fields at line 4250 below.